dev/breeze/src/airflow_breeze/utils/run_utils.py

```diff
@@ -460,7 +465,9 @@ def run_compile_www_assets(
         "--all-files",
         "--verbose",
     ]
-    get_console().print(f"[info] The output of the asset compilation is stored in: [/]{WWW_ASSET_OUT_FILE}\n")
+    get_console().print(
+        f"[info]The output of the asset compilation is stored in: [/]{WWW_ASSET_OUT_DEV_MODE_FILE}\n"
```

Comment:
If `run_compile_www_assets` is called with `dev=False`, we are going to print the `WWW_ASSET_OUT_DEV_MODE_FILE` for the output of the compilation ?

Reply:
aaaaaah. right :)